data/src/main/java/com/stop/data/repository/NearPlaceRepositoryImpl.kt

```diff
@@ -12,13 +14,21 @@ internal class NearPlaceRepositoryImpl @Inject constructor(
     override suspend fun getNearPlaces(
         version: Int,
         searchKeyword: String,
-        centerLon: Float,
-        centerLat: Float,
+        centerLon: Double,
+        centerLat: Double,
         appKey: String
-    ): List<Place> {
-        return nearPlaceRemoteDataSource.getNearPlaces(
-            version, searchKeyword, centerLon, centerLat, appKey
-        ).map { it.toUseCaseModel() }
+    ): Flow<List<Place>> = flow<List<Place>> {
```

Comment:
```suggestion
    ): Flow<List<Place>> = flow {
```
IDE 힌트에서 fow 변수의 타입 `<List<Place>>`를 명시하지 않아도 된다고 나오네요!

Reply:
0c4a797c7eee500921191d918d2b27f425450ec6